### DISCUSSION TITLE
Update on binaries for Windows64 for version 90+?

### DISCUSSION THREAD

**Discussion by Mysticodex** [Eloston/ungoogled-chromium#1643] [Q&A]
I don't yet see any binaries for Windows64, for Chromium version 90 or higher. Is this still on the roadmap?

Thanks

**Comment by calhix** [comment]
Not sure if you already found out but you can use https://chromium.woolyss.com/, just choose the install with the ungoogled flag. It won't auto-update of course so you should keep checking here.

**Reply by networkException** [reply]
I will say that woolyss' binaries are in no way affiliated with this project. Windows releases are planned but currently the project is simply lacking maintenance capacity for Windows at the moment.

**Comment by Mysticodex** [comment]
Thanks for the information, looking forward to the future of the project in
the Windows space. I personally think that it's an amazing alternate to
normal Chrome in the Windows environment, and have appreciated every
release.

Thanks again all!

**Comment by Mysticodex** [comment]
Running into an issue now, wherein newer extensions require version 90 minimum to run. Without 90 or higher releases, the extensions are stuck as older versions.

At this point in time, I am willing to compensate someone for their time to build the latest release binary of Ungoogled Chromium for Windows. I am not sure if this is allowed with the TOS of these kinds of open source projects, but please be in touch if you are willing you take on this job, so we can discuss compensation.

Thank you

**Comment by Esiou** [comment]
I just updated to version 95 through winget on windows. Special thanks to anyone who made this happen.

**Reply by tomasz1986** [reply]
> I just updated to version 95 through winget on windows. Special thanks to anyone who made this happen.

See https://github.com/microsoft/winget-pkgs/pull/32205. They just pull the pre-compiled binaries from https://github.com/macchrome/winchrome/releases.

**Comment by Mysticodex** [comment]
Just seeing this now, v95 for Win64 went up on Oct 30th? 

**Author: Nifury (view all releases from user)
Publication time (in UTC): 2021-10-30 08:31:05.010409+00:00
Installation instructions: (unspecified)
Notes: (none)**
